examples/plot_2_multidim_sigs.py

```diff
@@ -0,0 +1,249 @@
+"""
+2. 2D and 3D Signals
```

Comment:
I'd vote for a more descriptive title. Maybe "Running ByCycle on Multi-Dimensional Arrays"?